charts/cactus-backend/Chart.yaml

```
@@ -2,10 +2,11 @@ apiVersion: v2
 name: cactus-backend
 description: A backend that exposes an API for the Cactus app to use.
 type: application
-version: 0.5.4
+version: 0.5.5
 appVersion: "0.2"
 maintainers:
   - name: dataviruset
+  - name: shuning
```

```suggestion
  - name: shuninghuang
```
Hope you can get the `shuning` github username some day :P